src/components/learner-credit-management/cards/CourseCard.jsx

```diff
@@ -3,95 +3,144 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { camelCaseObject } from '@edx/frontend-platform';
-import cardFallbackImg from '@edx/brand/paragon/images/card-imagecap-fallback.png';
 import {
-  Badge, Button, Card, Hyperlink,
+  Badge,
+  Button,
+  Card,
+  Stack,
+  Hyperlink,
+  useMediaQuery,
+  breakpoints,
 } from '@edx/paragon';
-import { EXEC_COURSE_TYPE } from '../data/constants';
-import { formatDate } from '../data/utils';
+import { injectIntl } from '@edx/frontend-platform/i18n';
+
+import { CONTENT_TYPE_COURSE, EXEC_COURSE_TYPE } from '../data';
+import { formatCurrency, formatDate, getEnrollmentDeadline } from '../data/utils';
+import CARD_TEXT from '../constants';
+import defaultLogoImg from '../../../static/default-card-header-dark.png';
+import defaultCardImg from '../../../static/default-card-header-light.png';
```

Comment:
Done! We don't need a default logo?

Reply:
I don't feel we do, but could convinced otherwise. The `fallbackSrc` is important because without it, it messes up the card grid with one or more cards with differing heights without the image. Having a missing logo image doesn't affect the card heights in the same way, so having a fallback isn't as important.